models/CommentManager.php

```diff
@@ -0,0 +1,136 @@
+<?php
+
+class CommentManager extends Model{
+    
+    //FONCTIONS POUR LE FRONTEND
+    // Fonction qui permet de récupérer les commentaires visibles d'un article (seen = 1)
+     function get_comments_blog(){
+        $req = $this->db->query("SELECT * FROM comments WHERE article_id = '{$_GET['id']}' AND seen = 1 ORDER BY date DESC");
+
+        $responses=[]; // Obligé d'initialisé avant  la boucle sinon cela ne fonctionne pas.
+        
+        while($row = $req->fetch()){   
+            $response   = new Comment();
+            $response   ->setName($row['name']);
+            $response   ->setDate($row['date']);
+            $response   ->setComment($row['comment']);
+            $response   ->setArticleId($row['article_id']);
+            $response   ->setSeen($row['seen']);
+                        
+            $responses[] = $response;
+        }
+        return $responses;
+        }
+        // Fonction permettant de vérifier que les données envoyées pat l'utilisateur sont bonnes.
+        function form_comment_verification(){
+            if(filter_has_var(INPUT_POST, 'submit')){
+                $name = filter_var(htmlspecialchars(filter_input(INPUT_POST, 'name')));
+
+                $email = filter_var(htmlspecialchars(filter_input(INPUT_POST, 'email')));
+
+                $comment = filter_var(htmlspecialchars(filter_input(INPUT_POST, 'comment')));
+                $errors = [];
+
+                if(empty($name) || empty($email) || empty($comment)){
+                    $errors['empty'] = "Tous les champs n'ont pas été remplis";
+                }else{
+                    if(!filter_var($email, FILTER_VALIDATE_EMAIL)){
+                        $errors['email'] = "L'adresse email n'est pas valide";
+                    }
+                    if(strlen($name) < 5){
+                        $errors['name'] = "Votre nom doit contenir au moins 5 caractères.";
+                    }
+                    if(strlen($comment) < 5){
+                        $errors['comment'] = "Votre message doit contenir au moins 5 caractères.";
+                    }
+                }
+
+                if(!empty($errors)){
+                ?>
+                    <div class="card red">
+                        <div class="card-content white-text">
+                            <?php
+                                foreach($errors as $error){
+                                    echo $error."</br>";
+                                }
+                            ?>
+                        </div>
+                    </div>
+                <?php
+                }else{
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The method form_comment_verification uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them.](https://app.codacy.com/manual/JeromeLacquemant/projet-5/pullRequest?prid=5379676)